### PR TITLE
add shareit case study

### PR DESCRIPTION
**Pull request by dcalvin**
- [ ] I have added necessary CTA buttons
- [ ] I have resized the images to <= 200 KB
- [ ] I have included necessary SEO keywords in filenames

### CODE REVIEW

File: how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+---
+title: How SHAREit Powers Its AI Workflow with TiDB and TiKV
+author: ['Marshall Zhu']
+date: 2021-12-16
+summary: In this article, Marshall Zhu from SHAREit shares their AI workflow, the challenges they faced, and how two TiDB and TiKV power their AI platform and recommendation system.
```
Comment (septemberfd) [inline how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md:5]:
```suggestion
summary: In this article, Marshall Zhu from SHAREit shares their AI workflow, the challenges they faced, and how TiDB and TiKV power their AI platform and recommendation system.
```

File: how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+image: /images/blog/how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.png
+tags: ['AI', 'Feature Store', 'TiKV']
+customer: SHAREit Group
+customerCategory: Internet
+logo: /images/blog/customers/shareit-logo.png
+---
+
+**Industry:** Internet
+**Author:** Marshall Zhu (Big Data Director at SHAREit)
+**Editors:** Tom Dewan, Calvin Weng
```
Comment (septemberfd) [inline how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md:15]:
```suggestion
**Editors:** Tom Dewan, [Calvin Weng](https://github.com/dcalvin)
```

File: how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+## How does database power AI?
+
+Recall that before our AI workflow improvement effort, algorithm engineers spent 50% of their time on feature engineering and sampling. Now they spend 25% time (or less) on this task. Needless to say, they are a lot happier.
+
+Today, when the big data paradigm is clear, users are often affected by the empirical thinking of the scenario. However, after we divided the scenario into several demand points based on different quantity needs, we came up with our own new solutions. Now the usage of DB brings us some new angles. Single link latency is very important, and hopefully the interface can lay the foundation for subsequent modularization and automation..
+
+I hope you found this information helpful. If you face issues similar to ours, please consider investigating TiKV and TiDB.
+If you want to know more details about our story or have any questions, you're welcome to join the [TiDB community on Slack](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-blog) and send your feedback.
```
Comment (anotherrachel) [inline how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md:201]:
```suggestion
I hope you found this information helpful. If you face issues similar to ours, please consider investigating TiKV and TiDB.

If you want to know more details about our story or have any questions, you're welcome to join the [TiDB community on Slack](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-blog) and send your feedback.
```

File: how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
```
Comment (dcalvin) [inline how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md:201]:
```suggestion
I hope you found this information helpful. If you face issues similar to ours, please consider investigating TiKV and TiDB. If you want to know more details about our story or have any questions, you're welcome to join the [TiDB community on Slack](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-blog) and send your feedback.
```

**Review by anotherrachel — APPROVED** [review]

File: how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+
+* The first is to support multiple types at the proxy layer; for example, set, zSet, hash, and counter. We have made a lot of progress now.
+* In addition, as a storage role, KV interacts closely with the big data ecology, and in these scenarios, they often require batch input and batch export of data. Through this realization, we integrate TiKV/TIDB into the data & AI pipeline. Users don't have to worry about storage choices.
+
+## How does database power AI?
+
+Recall that before our AI workflow improvement effort, algorithm engineers spent 50% of their time on feature engineering and sampling. Now they spend 25% time (or less) on this task. Needless to say, they are a lot happier.
+
+Today, when the big data paradigm is clear, users are often affected by the empirical thinking of the scenario. However, after we divided the scenario into several demand points based on different quantity needs, we came up with our own new solutions. Now the usage of DB brings us some new angles. Single link latency is very important, and hopefully the interface can lay the foundation for subsequent modularization and automation..
```
Comment (septemberfd) [inline how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md:198]:
```suggestion
Today, when the big data paradigm is clear, users are often affected by the empirical thinking of the scenario. However, after we divided the scenario into several demand points based on different quantity needs, we came up with our own new solutions. Now the usage of DB brings us some new angles. Single link latency is very important, and hopefully the interface can lay the foundation for subsequent modularization and automation.
```

File: how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+
+The operator registration is currently being developed by the system engineer, because it is written in C++. The feature registration is developed by the data engineer, because it involves the complex modification of extract, transform, load (ETL) code.
+
+The sample generation process is very important and is common to every machine learning workflow. This process requires frequent communication, which requires that data engineers and algorithm engineers must work together closely.
+
+The model training and strategy development as well deployment required the same level of communication overhead. Our expanded workflow includes three service goals: feature extraction, inference, and feature monitor.
+
+The entire workflow conforms to the AI ​​paradigm. However, the core time-consuming steps are caused by insufficient standards and frequent interactions. Therefore, we divide every step into many processes to solve each problem separately. 
+
```
Comment (septemberfd) [inline how-shareit-powers-its-ai-workflow-with-tidb-and-tikv.md:91]: 麻烦加个 CTA 按钮吧~

**Pull request merged**